extensions/antd-open-browser/README.md

```diff
@@ -0,0 +1,3 @@
+# antd-open-browser
+
+快速打开 antd 组件
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.